.github/workflows/update.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  schedule:
+    - cron: '30 */6 * * *'
+  push:
+    branches:
+    - master
+name: update
+jobs:
+  update:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: checkout
+      uses: actions/checkout@master
+    - name: checkout reffy
+      uses: actions/checkout@master
+      with:
+        repository: tidoust/reffy
+        ref: master
+        fetch-depth: 1
+        path: reffy
+    - name: setup node
+      uses: actions/setup-node@master
+      with:
+        node-version: '10.x'
+    - name: install pandoc
+      run: sudo apt-get install pandoc
+    - name: setup reffy
+      run: |
+        echo "${{ secrets.CONFIG_JSON }}" | base64 --decode > config.json
+        npm ci
+      working-directory: ../reffy
+    - name: reffy crawl
+      run: npm run all
+      working-directory: ../reffy
+    - name: copy reports
+      run: rsync -av --exclude=README.md ../reffy/reports/ ./
```

Comment:
To address a question from the previous PR, this won't delete existing files, only add.